fhir-smart/src/test/java/com/ibm/fhir/smart/test/AuthzPolicyBackupEnforcementTest.java

```diff
@@ -0,0 +1,71 @@
+package com.ibm.fhir.smart.test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import com.ibm.fhir.config.FHIRRequestContext;
+import com.ibm.fhir.model.resource.Patient;
+import com.ibm.fhir.model.test.TestUtil;
+import com.ibm.fhir.persistence.interceptor.FHIRPersistenceEvent;
+import com.ibm.fhir.smart.AuthzPolicyEnforcementPersistenceInterceptor;
+
+public class AuthzPolicyBackupEnforcementTest {
+    AuthzPolicyEnforcementPersistenceInterceptor interceptor;
+    Patient patient;
+
+    @BeforeClass
+    public void setup() throws Exception {
+        interceptor = new AuthzPolicyEnforcementPersistenceInterceptor();
+
+        patient = TestUtil.readExampleResource("json/ibm/minimal/Patient-1.json");
+        patient = patient.toBuilder()
+                .id("11111111-1111-1111-1111-111111111111")
+                .build();
+    }
+
+    @Test
+    public void testUserInfoHeader() throws Exception {
```

Comment:
Negative test too?

Reply:
yes, needs much more comprehensive tests in general I think

Reply:
No problem - is this coming in a separate issue? Should we track an issue for it?

Reply:
I just posted another commit here...can you please have a look?